src/components/Login/Footer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { VStack, Button, HStack, Link, Text } from 'native-base';
+
+function Footer({ handleSubmit }) {
+  return (
+    <>
```

Comment:
Non-blocking: Let's make sure we're consistent as we get closer to releasing this. I see multiple styles for this "fragment" element. Let's stick with importing `{ Fragement }` and using `<Fragement>` to be consistent with the React JS project.